orcoursetrion/actions/github.py

```diff
@@ -72,12 +87,41 @@ def create_xml_repo(course, term, team, description=None):
         term=term
     )
     repo = github.create_repo(config.ORC_XML_ORG, repo_name, description)
-    github.add_team_repo(config.ORC_XML_ORG, repo_name, team)
+    # Add to the deployment team
```

Comment:
It still is, I just organized it differently so that put_team and add_team_repo go together for the course team, and the "deploy" team gets a comment treatment as well.